src/pages/index.js

```diff
@@ -63,13 +105,80 @@ const cardList = new Section(
 // Card create function
 
 function createCard(cardData) {
-  const card = new Card(cardData, "#card-template", (data) => {
-    imagePopup.open(data);
-  });
+  const card = new Card(
+    cardData,
+    "#card-template",
+    (data) => {
+      imagePopup.open(data);
+    },
+    handleDeleteCardClick,
+    handleLikeClick
+  );
   return card.getView();
 }
 
-//
+// Like Button
+
+function handleLikeClick(cardID, cardInstance) {
+  if (cardInstance._isLiked) {
+    api
+      .removeLike(cardID)
+      .then(() => {
+        cardInstance.setCardLike(false);
+        console.log("like removed for card ID", cardID);
+      })
+      .catch((error) => console.error("error removing like:", error));
+  } else {
+    api
+      .addLike(cardID)
+      .then(() => {
+        cardInstance.setCardLike(true);
+      })
+      .catch((error) => console.error("Error adding like", error));
+  }
+}
+
+// Delete Button
+
+function handleDeleteCardClick(cardInstance) {
+  confirmPopup.open();
+  confirmPopup.setSubmitAction(() => {
+    api
+      .deleteCard(cardInstance._id)
+      .then(() => {
+        cardInstance._handleDeleteCard();
```

Comment:
`_handleDeleteCard`

Please, delete the **low dash** (`underscore`) from methods/fields if you want to use them **outside** a class because the `low dash` shows a **private** field which is used only inside the class.